src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post" onsubmit="onSubmit()"
+    >
+      <fieldset name="Personal information">
+        <legend>Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input
+          type="text"
+          id="surname"
+          required>
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input
+          type="text"
+          id="name"
+          required
+          >
+        </label>
+
+        <label class="form-field" for="age">
+          How old are You?
+          <input
+           type="number"
+           id="age"
+           min="12"
+           max="100"
+           value="12"
+           required
+           >
+        </label>
+
+        <label class="form-field" for="birth">Full date of birth:
```

Comment:
Перенось на новий рядок
```suggestion
        <label class="form-field" for="birth">
          Full date of birth:
```